src/main/java/uk/gov/ons/ctp/integration/rhsvc/event/impl/UACEventReceiverImpl.java

```diff
@@ -26,6 +28,9 @@
   @Autowired private RespondentDataRepository respondentDataRepo;
   private static final Logger log = LoggerFactory.getLogger(UACEventReceiverImpl.class);
 
+  @Value("#{'${queueconfig.qid-filter-prefixes}'.split(',')}")
```

Comment:
The QIDs would be stored in application.yml using list notation, then this is unnecessary.